js/main.js

```diff
@@ -0,0 +1,56 @@
+var $p = document.querySelector('p.inactive');
+var $webPage = document.querySelector('body');
+var currentCharacter = 0;
+
+function gameLoading(event) {
+  var xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://animechan.vercel.app/api/random');
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function () {
+    var $anime = document.querySelector('h3.anime');
+    var $character = document.querySelector('h3.character');
+    $anime.textContent = `Anime: ${xhr.response.anime}`;
+    $character.textContent = `Character: ${xhr.response.character}`;
+    var wordList = xhr.response.quote.split(' ');
+    for (var i = 0; i < wordList.length; i++) {
+      var $newWord = document.createElement('word');
+      for (var wordIndex = 0; wordIndex < wordList[i].length; wordIndex++) {
+        var $letter = document.createElement('letter');
+        $letter.textContent = wordList[i][wordIndex];
+        $newWord.appendChild($letter);
+      }
+      var $space = document.createElement('letter');
```

Comment:
What are `word` and `letter` elements? You shouldn't ever make entirely custom element types, it destroys SEO. You should use a `span` with a class applied to it.